pkg/os/shell/shell_windows.go

```diff
@@ -1,5 +1,90 @@
 package shell
 
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
 var (
 	supportedShell = []string{"cmd", "powershell"}
 )
+
+// re-implementation of private function in https://github.com/golang/go/blob/master/src/syscall/syscall_windows.go
```

Comment:
https://github.com/mitchellh/go-ps can probably be used if we don't want to c&p this code directly (but not expecting any changes related to this just now)